go.mod

```diff
@@ -7,24 +7,26 @@ require (
 	github.com/BurntSushi/toml v0.4.1
 	github.com/containers/libtrust v0.0.0-20190913040956-14b96171aa3b
 	github.com/containers/ocicrypt v1.1.2
-	github.com/containers/storage v1.35.0
+	github.com/containers/storage v1.36.3
```

Comment:
For the record, this is a .Y version bump, but we are doing it for Buildah 1.23.Z, which already uses c/storage 1.36.0, and for Podman 3.4.2-rhel, which uses 1.36.1.

Is that correct?

Reply:
I think those changes are fine (though ultimate Buildah/Podman maintainers might have a different opinion) — it’s just two targeted fixes.